mcs/class/corlib/System/Guid.MonoTouch.cs

```diff
@@ -1,5 +1,3 @@
-#if MONOTOUCH && FULL_AOT_RUNTIME
```

Comment:
We should rename the file

Reply:
How about `Guid.CommonCrypto.cs`?

Reply:
I moved it into the `CommonCrypto/` directory and renamed it into `CommonCrypto/Guid.cs`.